packages/block-editor/src/components/inserter/menu.js

```diff
@@ -316,6 +316,7 @@ function InserterMenu(
 		>
 			<div className="block-editor-inserter__main-area">
 				<TabbedSidebar
+					className="block-editor__tabbed-sidebar"
```

Comment:
We need this so that we can set a width for only the inserter a the list view.

Reply:
This ends up compiling to `block-editor-tabbed-sidebar block-editor__tabbed-sidebar` which feels like a workaround. How about one of these options:

- Give it a name that describes the 350 width, like: `block-editor-tabbed-sidebar--wide`
- Name it as the inserter and list view, i.e. `block-editor-inserter__tabbed-inserter`
- Use the className above it to nest the CSS.